### ISSUE TITLE
Viewing a dataframe after installing pandas crashes

### ISSUE THREAD

**Issue by akdor1154** [spyder-ide/spyder#9986]
### What steps will reproduce the problem?

1. Install Spyder, don't install pandas.
2. Go to view a pandas dataframe (e.g. if your project python executable is in a venv)
3. Get an error that pandas isn't installed.
4. Realize you haven't got pandas installed globally alongside Spyder, so install it with pip.
5. Go to view the pandas dataframe again.
6. See this crash.

### Traceback
```python-traceback
  File "/home/jarrad/.local/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/objectexplorer/tree_model.py", line 267, in fetchMore
    parent_item.obj_path)
  File "/home/jarrad/.local/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/objectexplorer/tree_model.py", line 322, in _fetchObjectChildren
    for attr_name, attr_value in sorted(inspect.getmembers(obj)):
  File "/usr/lib/python3.7/inspect.py", line 341, in getmembers
    value = getattr(object, key)
  File "/home/jarrad/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 391, in _constructor_expanddim
    raise NotImplementedError("Not supported for DataFrames!")
NotImplementedError: Not supported for DataFrames!
Traceback (most recent call last):
  File "/home/jarrad/.local/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/objectexplorer/tree_model.py", line 267, in fetchMore
    parent_item.obj_path)
  File "/home/jarrad/.local/lib/python3.7/site-packages/spyder/plugins/variableexplorer/widgets/objectexplorer/tree_model.py", line 322, in _fetchObjectChildren
    for attr_name, attr_value in sorted(inspect.getmembers(obj)):
  File "/usr/lib/python3.7/inspect.py", line 341, in getmembers
    value = getattr(object, key)
  File "/home/jarrad/.local/lib/python3.7/site-packages/pandas/core/frame.py", line 391, in _constructor_expanddim
    raise NotImplementedError("Not supported for DataFrames!")
NotImplementedError: Not supported for DataFrames!
```

## Versions

* Spyder version: 4.0.0b4 
* Python version: 3.7.3
* Qt version: 5.12.2
* PyQt5 version: 5.12.1
* Operating System: Linux 5.0.0-20-generic

### Dependencies

```
pygments >=2.0    :  2.3.1 (OK)
qdarkstyle >=2.6.4:  2.7 (OK)
sphinx >=0.6.6    :  2.1.2 (OK)
psutil >=0.3      :  5.5.1 (OK)
pyls >=0.27.0     :  v0.28.1 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  None (OK)
numpy >=1.7       :  1.16.2 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.5.2 :  4.5.2 (OK)
IPython >=4.0     :  5.8.0 (OK)
matplotlib >=2.0.0:  3.0.2 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
> Realize you haven't got pandas installed globally alongside Spyder, so install it with pip.

You need to restart Spyder for this change to take effect.

**Comment by ccordoba12** [comment]
@dalthviz, was this fixed by PR #9967?

**Comment by dalthviz** [comment]
@ccordoba12 I will test but technically yes, this should be fixed by that PR (since there we made a change to directly handle the errors of the `getattr` call per attribute and skip the attribute if we can't get its value).

**Comment by ccordoba12** [comment]
Ok, please test it and if this is fixed, please remove the Milestone and close this one.

**Comment by dalthviz** [comment]
This is fixed by PR #9967
